src/main/groovy/de/dkfz/roddy/execution/jobs/BatchEuphoriaJobManager.groovy

```diff
@@ -307,11 +307,7 @@ abstract class BatchEuphoriaJobManager<C extends Command> {
         BEJobResult jobResult
         if (res.successful) {
             String exID
-            try {
-                exID = parseJobID(res.resultLines.join("\n"))
-            } catch (BEException ex) {
-                throw new BEException("Full input was: '${res.resultLines.join("\n")}", ex)
```

Comment:
Why did you remove that? Now there is less information showing why this failed. I specifically added this, because with the full output you can directly see what is going wrong. If that is the wrong thing for Roddy (e.g. because you think the user should not get flooded with output), then move the BEException into a RoddyException. But within BatchEuphoria -- no matter what we need outside -- the full output is the right thing.

Reply:
I cannot remember that I did remove it. What does the code annotation say? My actual changes are only in DirectCommand. Maybe this happened because we work on 0.0.3?

Reply:
@fkaercher changed this, apparently already in December! No clue how this ends up in your PR? I checked the subclasses and there is an XMLSlurper used in one. In any case, the parser can potentially throw, and I thing that putting the complete (failed to parse) parsed output in the exception seems to be the right way to do it. What do you think about this? We could change it back ...